deploy/backplane/cssre/rhoc/02-rhoc-cssre-admins-cluster.SubjectPermission.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-cssre-rhoc-admins-project
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: backplane-cssre-rhoc-project
+    namespacesAllowedRegex: (^redhat-openshift-connectors$)
```

Comment:
Just checking here - I see some `SubjectPermission` RBAC elsewhere defined in MCC that gets applied to `redhat-openshift-connectors.*` (ie. with the tailing wildcard) and just wanted to check whether that needs to be set here as well.

Reply:
I know the RHOC team have their own permission set if that is what you are referring to; if so, yes we would need it here as well, as the assumption would be that the Engineering team will possibly be removing these permissions at some point.